pulpcore/cli/core/task.py

```diff
@@ -112,3 +114,33 @@ def cancel(
             if not re.match("Task /pulp/api/v3/tasks/[-0-9a-f]*/ canceled", str(e)):
                 raise e
         click.echo(_("Done."), err=True)
+
+
+DATETIME_FORMATS = ["%Y-%m-%d", "%Y-%m-%dT%H:%M:%S"]
+
+
+@task.command()
+@click.option(
+    "--finished-before",
+    "finished",
+    help=_("Purge task-records whose 'finished' time is **before** the time specified."),
+    type=click.DateTime(formats=DATETIME_FORMATS),
```

Comment:
Is click injecting the formatting info into the help text?

Reply:
Yes it is